packages/features/insights/components/AverageEventDurationChart.tsx

```diff
@@ -18,8 +19,8 @@ export const AverageEventDurationChart = () => {
   const initialConfigIsReady = !!(initialConfig?.teamId || initialConfig?.userId || initialConfig?.isAll);
   const { data, isSuccess, isPending } = trpc.viewer.insights.averageEventDuration.useQuery(
     {
-      startDate: startDate.toISOString(),
-      endDate: endDate.toISOString(),
+      startDate: dayjs(startDate).utc(true).toISOString(),
```

Comment:
I don't think going .utc(true) is a good idea here - utc() is better suited. it comes in (startDate) as UTC too, however you could simplify then by going dayjs.utc(startDate).